vault/core/_audit

```diff
@@ -0,0 +1 @@
+{"Value":"AAAAAQLqm3nyg2QzB/a+ZnYay7oGLXCUkDHOoH6pV+EtV2PNzl3RdgZevQQaFZPgxwsz0kaVpMMI0/PWJN54+li2xVWHzFLEr6TAFWQh94DVo4D6GKcIeExh"}
```

Comment:
ideally, vault should be outside blockchain-automation-framework, but you have now added gitignore